docs/custom-features/scheduled-jobs.md

```diff
@@ -11,9 +11,16 @@ every minute, and [`publish_scheduled_without_bundles`](https://github.com/ONSdi
 
 ## `publish_bundles`
 
-Is a management command that publishes [bundles](bundles.md) that are scheduled and ready to publish.
+Is a management command that publishes [bundles](bundles.md) that are scheduled and ready to publish. Bundles are considered for publishing if their release date is in the past.
+
+### `--include-future`
+
+To reduce the impact of the process and infrastructure warm-up time, `publish_bundles` starts earlier than necessary and includes bundles which need to be published in the future. The command then waits until each bundle's release time, and then publishes it. Bundles due to be published in the past are still published immediately without wait.
```

Comment:
Non-blocking nit-pick: I understand you likely mean `wait()` here but I think this just might read better.
```suggestion
To reduce the impact of the process and infrastructure warm-up time, `publish_bundles` starts earlier than necessary and includes bundles which need to be published in the future. The command then waits until each bundle's release time, and then publishes it. Bundles due to be published in the past are still published immediately without waiting.
```